packages/accessible-react-app/src/js/components/PrimaryNav.js

```diff
@@ -3,12 +3,13 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 
 const PrimaryNav = props => {
-  const { ariaLabel } = props;
+  const { ariaLabelText } = props;
 
   return (
-    <nav aria-label={ariaLabel}>
-      {/* FE Talking Point - Unique aria-labels for repeated blocks like <nav> */}
-      <ul className="ac-primary-nav">
+    <nav aria-label={ariaLabelText}>
+      {/* SR Talking Point - Unique aria-labels for repeated blocks like <nav> */}
+      {/* eslint-disable-next-line jsx-a11y/no-redundant-roles */}
+      <ul className="ac-primary-nav" role="list">
```

Comment:
role="list" should be the default so we shouldn't need it hard coded. Was it being overwritten somewher?